tests/test_api/test_worker.py

```diff
@@ -87,4 +80,35 @@ def testApiWorker_Distributor(tmpDir, tmpConf, monkeypatch):
     assert status is False
     assert valid is False
 
-# END Test testApiWorker_Distributor
+# END Test testCoreWorker_Distributor
+
+# validate tests
+@pytest.mark.api
+def testCoreWorker_validator_fails(monkeypatch):
+    """Test the Worker class validator.
+    """
+    current_folder = os.path.dirname(__file__)
+    fn = os.path.join(current_folder,
+            '../files/api/aqua-modis-1km-20210429002844-20210429003955.xml')
+    data = readFile(fn)
+    def mock_check_information_content(*args, **kwargs):
+        return True, ""
+    monkeypatch.setattr(Worker, '_check_information_content', mock_check_information_content)
```

Comment:
I would prefer it if we could keep one test per function, and instead split up mocking by using the monkeypatch context facility to test different scenarios. It makes the code and test output a lot less verbose.

Reply:
ok - I changed the code. Is it better?

Reply:
I can't see any change in the code here

Reply:
I'm not sure if I understand - one test per function meaning one test-function per function that should be tested, or one test-function per test (i.e., per assertion)? Also, context facility and monkeypatching is new to me, and I didn't take time for reading up... It does the job, doesn't it? ;-)